### ISSUE TITLE
Zeitwerk::Loader.eager_load_all fails in rake db:seed context

### ISSUE THREAD

**Issue by Andrew-Max** [rails/rails#37851]
Not sure if this is a bug but it seems possible.

### Steps to reproduce
at `/app/models/coordinates_modifier/circular_to_linear_actuator.rb`
```ruby
class CoordinatesModifier::CircularToLinearActuator < CoordinatesModifier
end
```

```ruby
class CoordinatesModifier < ApplicationRecord
  include SingleTableInheritable

  belongs_to :action_coordinates
end
```

at `/db/seeds.rb`
```ruby
 CoordinatesModifier::CircularToLinearActuator.new
```

throws the error `NameError: uninitialized constant #<Class:0x0000560b89165950>::CircularToLinearActuator
`

Additionally attempting to manually load everything with Zeitwerk fails

```bash
[4] pry(main)> Zeitwerk::Loader.eager_load_all
NameError: uninitialized constant ActionMailbox::InboundEmail(Table doesn't exist)::Incineratable
Did you mean?  ActionMailbox::IncinerationJob
````

### Expected behavior
In Rails 5 these objects were available as is. My assumption is that they should get loaded according to the same rules that govern normal rails env initialization. 
### Actual behavior
Classes are not available and `Zeitwerk::Loader.eager_load_all` fails
### System configuration
**Rails version**:
6.0.1
**Ruby version**:
2.6.3

**Comment by fxn** [comment]
Can you please paste the definition of `CoordinatesModifier`?

**Comment by fxn** [comment]
I cannot reproduce using Rails 6.0.1 and ruby 2.6.5p114. Could please upload a minimal application where these errors happen?

**Comment by Andrew-Max** [comment]
@fxn creating the reproduction case indicated that maybe there is a bug here but it is definitely different than what I thought it was.

The failure to find the class here only occurs if I have a binding.pry before the first mention of the class.

so at the beginning of the seed file if I bind at: 
```ruby
binding.pry
CoordinatesModifier::CircularToLinearActuator.create
```
at that frame, referencing the class will throw the error
`NameError: uninitialized constant #<Class:0x000055eae5eb7d70>::CircularToLinearActuator`

I'm assuming I should close this then? If you think that sounds like a valid separate bug I'd be happy to throw up  a new issue for it.

**Comment by fxn** [comment]
Ah, yes. This is a known incompatibility between `byebug` and `zeitwerk` documented [here](https://github.com/fxn/zeitwerk#autoloading-explicit-namespaces-and-debuggers).

@gsamokovarov is working on a [new debugger](https://github.com/gsamokovarov/break) that does not have this limitation, but I don't know which is its status... how's that going man :).

**Comment by fxn** [comment]
@Andrew-Max I've played around with [`break`](https://github.com/gsamokovarov/break) and worked well for me. Would you like to give it a try?

**Comment by Andrew-Max** [comment]
@fxn I gave break a quick run. It looks like a great project but I think it was just generally not playing nice with my app or I did not initialize correctly. I was seeing the same issues more or less but I wasn't really able to get much debugging functionality out of it at all so I would say I could neither confirm nor deny.

I'm going to close this out though. As long as I'm aware of where I'm throwing my `binding.pry`s I can make this setup work as is. 

Thanks for the attentive help.
